pkg/assert/validate.go

```diff
@@ -80,18 +86,35 @@ func runAssertion(
 // ValidateOnHost runs an exec function on RunCommandHost and assert given is fulfilled.
 // The last argument should be the assertion.
 // Need to send kubeconfig file.
-func ValidateOnHost(args ...string) error {
+func ValidateOnHost(opt AsyncOpt, args ...string) error {
```

Comment:
Do we need this?

Reply:
We need a way when call this function to pass a timeout customized because not every caller needs the same amount of time. For instance we have functions on the processor code that needs really small times, and other places that needs more ( like 420 s that today is hard-coded)